src/index.html

```diff
@@ -391,8 +391,8 @@ <h5>Formats</h5>
               <p>The license to this software is bundled with the application and is also <a integrity="sha512-/VmsC2zX+w/AMx3VBBib2r3ICkFlHUUfRDxxdF593TGHc2D3oWgaHmZsXhQdXR5iMJAS+DGQ8YaCAiFzXu+sWQ==" href="https://github.com/openculinary/frontend/blob/b2bcab413b424d9b6e1a64db16777beccdfb6ba7/LICENSE">available to read online.</a></p>
               <p>Included below are the licenses for the dependencies of this application.</p>
               <!-- TODO: Use webpack build to embed integrity hashes for 'licenses.txt'; see https://github.com/waysact/webpack-subresource-integrity/issues/208 -->
-              <iframe integrity="sha512-+6cx6o7EBi008aQgVnWtwy1ZCZA0pidal0f5Du+Ss0Fj9oDu06jdem6tHkUB3ZWWtasVKqL/HAbFdY7IVc9PAA==" class="container-fluid" src="licenses.txt"></iframe>
-              <p>The license for the packages that this application depends upon are <a integrity="sha512-O1Mh4TtDC6z+cTZsRkE9cTmpxAVcFqffyQ7Ck5vYK84h6b3up0aRePLeiuyHIoxR8MxdGRhP/Gn4uDTUmqjhhQ==" href="licenses.txt">available to read here.</a></p>
+              <iframe integrity="sha512-OiBpoj+pAz1+xEGHjWSgFSDfmzypvkr1oveSJTkUE+CUCGNeUhQq5a1YRVVxcOj+d9QaIPrH/NO56cxBSgtkgA==" class="container-fluid" src="licenses.txt"></iframe>
+              <p>The license for the packages that this application depends upon are <a integrity="sha512-OiBpoj+pAz1+xEGHjWSgFSDfmzypvkr1oveSJTkUE+CUCGNeUhQq5a1YRVVxcOj+d9QaIPrH/NO56cxBSgtkgA==" href="licenses.txt">available to read here.</a></p>
```

Comment:
It's probably worth adding a step to the `license-check` target in the `Makefile` to confirm that the `licenses.txt` integrity hash appears in `index.html`.